libvcx/src/api/vcx.rs

```diff
@@ -644,17 +692,29 @@ mod tests {
     fn test_open_wallet_with_wrong_name_fails() {
         let _setup = SetupDefaults::init();
 
-        let (export_wallet_path, wallet_name) = create_main_wallet_and_its_backup();
+        let (export_wallet_path, wallet_name, wallet_config) = create_main_wallet_and_its_backup();
 
-        wallet::delete_wallet(&wallet_name, settings::DEFAULT_WALLET_KEY, settings::WALLET_KDF_RAW, None, None, None).unwrap();
+        wallet::delete_wallet(&wallet_config).unwrap();
 
-        let import_config = json!({
-            settings::CONFIG_WALLET_NAME: wallet_name.as_str(),
-            settings::CONFIG_WALLET_KEY: settings::DEFAULT_WALLET_KEY,
-            settings::CONFIG_WALLET_KEY_DERIVATION: settings::WALLET_KDF_RAW,
-            settings::CONFIG_EXPORTED_WALLET_PATH: export_wallet_path.path,
-            settings::CONFIG_WALLET_BACKUP_KEY: settings::DEFAULT_WALLET_BACKUP_KEY,
-        }).to_string();
+        let wallet_name = &format!("export_test_wallet_{}", uuid::Uuid::new_v4());
+        let wallet_config2 = WalletConfig {
```

Comment:
Why not `wallet_config`? I don't see the need to introduce another variable name

Reply:
renamed